devstack/clean.sls

```diff
@@ -32,20 +32,17 @@ openstack-devstack cleandown:
   user.absent:
     - name: {{ devstack.local.username }}
     - purge: True
-    - onlyif: getent passwd {{ devstack.local.username }}
   cmd.run:
```

Comment:
Isn't this `cmd.run` redundant because of the `user.absent` `purge:true`?

Reply:
Yes it's always redundant unless `user.absent `state fails causing cmd.run (onfail) to be triggered.  I recall adding cmd.run due to some unexpected results from user.absent (possibly related to fact this server uses Active Directory, or an environment problem).

Reply:
Ah, gotcha. I thought it was, perhaps, due to the fact that the user was still in use (where a `force in user.absent` would help?).

Reply:
Ahh to have the time to figure out cause of every problem ;-)   Adding `force: true` is no harm, added!! thanks for the review comments.